packages/graph-editor/src/redux/selectors/graph.ts

```diff
@@ -9,6 +9,8 @@ export const logSelector = createSelector(graph, (state) => state.logs);
 
 export const playStateSelector = createSelector(graph, (state) => state.graphPlayState);
 
+export const mainGraphSelector = createSelector(graph, (state) => state.panels?.graph1);
```

Comment:
Moving forward this feels too brittle

Reply:
We were hardcoding graph id everywhere. I added a constant and used it